libdevcore/CommonData.h

```diff
@@ -172,9 +179,9 @@ inline std::string toHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd)
 	return (prefix == HexPrefix::Add) ? "0x" + str : str;
 }
 
-inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, unsigned _min = 0)
+inline std::string toCompactHex(u256 val, HexPrefix prefix = HexPrefix::DontAdd, PrintZero _min = PrintZero::DontPrint)
```

Comment:
This looks bad. Can you describe what the use cases for this function are? If tests needs something different maybe it will be better for `dev::test` to have a separated function for this case.